package.json

```diff
@@ -83,6 +92,8 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-test-loader": "^2.2.0",
+    "ember-cli-typescript": "^2.0.0-beta.2",
+    "ember-cli-typescript-blueprints": "^2.0.0-beta.1",
```

Comment:
These are very recently-cut beta releases. I leave it to the ember-data maintainers to decide whether their apptetite for risk aligns for merging this PR before a stable v2 release is cut

Reply:
idk. generating tests might be pretty useful